rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -350,7 +350,12 @@ type SuspendedReason = variant {
     Quarantined;
 };
 
-type Event = variant {
+type Event = record {
```

Comment:
Here for a lack of a better place: Could you add in the PR description why `CI_OVERRIDE_DIDC_CHECK` is necessary and why breaking the Candid compatibility here is acceptable (because the breaking changes only concern a debug endpoint)